relay/nakama/allowlist.go

```diff
@@ -132,41 +132,41 @@ type ClaimKeyRes struct {
 	Success bool `json:"success"`
 }
 
-func claimKeyRPC(ctx context.Context, _ runtime.Logger, _ *sql.DB, nk runtime.NakamaModule, payload string) (
+func claimKeyRPC(ctx context.Context, logger runtime.Logger, _ *sql.DB, nk runtime.NakamaModule, payload string) (
 	string, error,
 ) {
 	userID, err := getUserID(ctx)
 	if err != nil {
-		return "", err
+		return logCode(logger, Internal, "unable to get userID")
 	}
 
 	// if this user is already verified,
 	err = checkVerified(ctx, nk, userID)
 	if err == nil {
-		return "", eris.Wrap(errors.Join(ErrAlreadyVerified, err), "")
+		return logCode(logger, AlreadyExists, "user already verified with beta key")
 	}
 
 	var ck ClaimKeyMsg
 	err = json.Unmarshal([]byte(payload), &ck)
 	if err != nil {
-		return "", eris.Wrap(err, "")
+		return logCode(logger, Internal, "unable to unmarshal payload")
```

Comment:
The client is the source of the payload, correct? This looks like an InvalidArgument error. And the contents of error should be returned to the user.

Reply:
yep, changed to InvalidArgument